contracts/BondingClaimer.sol

```diff
@@ -0,0 +1,23 @@
+//SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+import {IBondingClaimer} from "./interfaces/IBondingClaimer.sol";
```

Comment:
For better readability I suggest we add spaces before brackets: { IBondingClaimer }